aws/resource_aws_ebs_encryption_by_default.go

```diff
@@ -0,0 +1,80 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsEbsEncryptionByDefault() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEbsEncryptionByDefaultCreate,
+		Read:   resourceAwsEbsEncryptionByDefaultRead,
+		Update: resourceAwsEbsEncryptionByDefaultUpdate,
+		Delete: resourceAwsEbsEncryptionByDefaultDelete,
```

Comment:
If we are going to do nothing in the `Delete` function, this can use `schema.Noop` instead of a custom function. 

```suggestion
		Delete: schema.Noop,
```

The documentation in that case should also explicitly state that it is only removing Terraform's management of the setting.

Instead of an empty `Delete` function though, I think a better user experience would be do disable encryption when this resource is removed. 👍